src/sage/functions/bessel.py

```diff
@@ -590,15 +610,16 @@ def _evalf_(self, n, x, parent=None, algorithm=None):
         """
         EXAMPLES::
 
-            sage: bessel_Y(0.5, 1.5)
+            sage: bessel_Y(0.5, 1.5)                                                    # needs mpmath
             -0.0460831658930974
```

Comment:
Is it right that this need `mpmath` but the above with input `(0,0)` or `(1,0)` does not?

Reply:
That seems to depend on the type of the input (symbolic or numerical)...

Reply:
Yes, exact inputs such as integer and rationals give symbolic output. The presence of inexact (RR) inputs triggers the numerical evaluation using mpmath.